modules/@apostrophecms/page-type/index.js

```diff
@@ -376,6 +376,9 @@ module.exports = {
   },
   extendMethods(self) {
     return {
+      enableAction() {
```

Comment:
@boutell as far as I could tell a custom page's `MODULE.action` was not actually useful, we often hard code in an action of `@apostrophecms/page` to do things like get documents.

In a consult w @abea some version of this seemed reasonable

Reply:
Later we could probably clean up some other code with this available.